debian/cloud-init.postinst

```diff
@@ -390,23 +216,6 @@ EOF
    # fix issue where cloud-init misidentifies the location of grub
    fix_1336855
 
-   # make upgrades disable network changes by cloud-init
-   disable_network_config_on_upgrade "$2"
-
-   remove_deprecated_cloud_config_modules "$2"
-
-   fix_azure_upgrade_1611074 "$2"
-
-   cleanup_ureadahead "$2"
-   fix_lp1889555 "$2"
-   change_cloud_init_output_log_permissions "$2"
-
-   rename_hook_hotplug_udev_rule
```

Comment:
Did you mean to include this one? The definition hasn't been deleted and this is a relatively recent addition.

Reply:
No I didn't - fixed.

Reply:
I'm not seeing it.